src/test/java/org/opensearch/ad/ml/CheckpointDaoTests.java

```diff
@@ -1010,7 +1011,8 @@ private double[] getPoint(int dimensions, Random random) {
     // The checkpoint used for this test is from a single-stream detector
     public void testDeserializeRCFModelPreINIT() throws Exception {
         // Model in file 1_3_0_rcf_model_pre_init.json not passed initialization yet
-        String filePath = getClass().getResource("1_3_0_rcf_model_pre_init.json").getPath();
```

Comment:
This code won't work on windows?

Reply:
See PR description

Reply:
Yeah it doesn't, the reason is because `.getPath()` adds a `/` to beginning of path which is fine for linux but not for windows. It causes `java.nio.file.InvalidPathException: Illegal char <:> at index 2`

Reply:
I'm not sure, as ml-commons also using `getClass().getResource` and it passed windows Github CI , check code https://github.com/opensearch-project/ml-commons/blob/2.x/ml-algorithms/src/test/java/org/opensearch/ml/engine/algorithms/text_embedding/TextEmbeddingModelTest.java#L90

Reply:
I think it's fine to use either way. Just curious why AD can't work.

Reply:
ML-Commons uses `toUri()` not `getPath(). Thats what I changed to using. `toUri()` is platform independent

Reply:
oh, got it, I should read it more carefully. Thanks, that make sense.